Lib/shutil.py

```diff
@@ -446,7 +446,7 @@ def copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
     else:
         ignored_names = set()
 
-    os.makedirs(dst)
+    if not os.path.isdir(dst): os.makedirs(dst)
```

Comment:
There is an `exist_ok` flag for `os.makedirs` but you should really make a bugs.python.org issue for this so that it can be discussed. Especially given that it breaks existing behavior.

Reply:
you're right, maybe adding "exist_ok" (or "dir_exists_ok") as an option to "copytree" is a better way fwd?